tests/user_input_checks_test.py

```diff
@@ -18,14 +15,7 @@
     process_simulator,
     process_x_o,
 )
-from sbi.user_input.user_input_checks_utils import (
-    CustomPytorchWrapper,
```

Comment:
Oh, it seems some of the tests got lost during my last refactor. There should be tests for the wrappers as well. I will look them up and add them here in this PR, ok?

Reply:
I just saw unused imports and killed them, perhaps that's what this is?

Reply:
yes, exactly. You can leave it like it is, I will fix this in a separate single commit later.